portfolio/src/main/webapp/src/util/types.ts

```diff
@@ -0,0 +1,2 @@
+export type Thunk<T> = T | (() => T);
```

Comment:
I think these generic types will become unnecessary once you have file level types that are less generic in html.ts

Reply:
I ended up not needing `Thunk<T>`, but I still find `Arrunk<T>` useful. Fixed in #10